compute_tools/src/extension_server.rs

```diff
@@ -158,14 +159,14 @@ fn parse_pg_version(human_version: &str) -> PostgresMajorVersion {
 pub async fn download_extension(
     ext_name: &str,
     ext_path: &RemotePath,
-    remote_ext_base_url: &str,
+    remote_ext_base_url: &Url,
```

Comment:
I think it makes sense to combine this to into one thing if this is possible.

@tristan957 WDYT?

Reply:
Okay, the `RemotePath` is a different thing from what I expected, so this is not feasible using the existing types.

Reply:
whatever you want!